test/box/net.box_iproto_transactions_over_streams.result

```diff
@@ -1004,26 +851,9 @@ test_run:cmd("start server test with args='10, true'")
 server_addr = test_run:cmd("eval test 'return box.cfg.listen'")[1]
 ---
 ...
-test_run:switch("test")
----
-- true
-...
-s1 = box.schema.space.create('test_1', { engine = 'memtx' })
+cnt = create_remote_spaces({"test_1", "test_2"}, {"memtx", "vinyl"})
```

Comment:
Please don't test vinyl here. All multi-engine tests should live in test/engine. Anyway, iproto streams have nothing to do with vinyl so I don't think you need to create vinyl spaces here.

Reply:
fixed